deps/build.jl

```diff
@@ -78,4 +111,11 @@ copy_config("logo-32x32.png", juliakspec)
 copy_config("logo-64x64.png", juliakspec)
 
 eprintln("Installing julia kernelspec $spec_name")
-run(`$jupyter kernelspec install --replace --user $juliakspec`)
+run(`$jupyter-kernelspec install --replace --user $juliakspec`)
```

Comment:
Instead of circumventing jupyter/notebook#448 we could use IPython 3 by doing `Conda.add("ipython=3.*")` until jupyter/notebook#448 is closed.